controllers/operator/migration/migration.go

```diff
@@ -766,15 +766,15 @@ func insertZenInstanceUsers(ctx context.Context, mongodb *MongoDB, postgres *Pos
 			continue
 		}
 		args := pgx.NamedArgs{
-			"UZID":          zenInstanceUser.UZID,
+			"UID":           zenInstanceUser.UID,
 			"UserID":        zenInstanceUser.UserID,
 			"ZenInstanceID": zenInstanceUser.ZenInstanceID,
 		}
 
 		query := `
 			INSERT INTO platformdb.zen_instances_users
-			(uz_id, zen_instance_id, user_id)
-			VALUES (@UZID, @ZenInstanceID, @UserID)
+			(uid, zen_instance_id, user_id)
+			VALUES (@UID, @ZenInstanceID, @UserID)
```

Comment:
This will not work - `_id` in ZenInstanceUsers is a MongoDB ObjectId, but the DDL has changed this to be a UUID. To do this, you'll need to swap `@UID` for `DEFAULT`.

Reply:
@rwhundley , Thank you for pointing this, I just correct it.
replaced `@UID` with `DEFAULT`